sql/mob_groups.sql

```diff
@@ -13475,59 +13475,59 @@ INSERT INTO `mob_groups` VALUES (71,3247,215,'Qiqirn_Bewitcher',0,128,0,0,0,0,0,
-INSERT INTO `mob_groups` VALUES (36,2819,216,'Nehebkau',0,128,1777,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (37,293,216,'Avalerion',0,128,194,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (38,2195,216,'Karkatakam',0,128,1425,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (39,2905,216,'Nonno',0,128,1822,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (40,4056,216,'Tuskertrap',0,128,2503,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (41,2918,216,'Npfundlwa',0,128,1826,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (42,729,216,'Cirein-croin',0,128,471,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (43,108,216,'Amhuluk',0,128,70,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (44,3695,216,'Sobek',0,128,2290,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (45,2094,216,'Ironclad_Pulverizer',0,128,1371,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (46,1842,216,'Gukumatz',900,0,1253,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (47,256,216,'Asanbosam',900,0,175,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (48,1932,216,'Heqet',900,0,1301,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (49,25,216,'Abyssic_Cluster',900,0,5,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (50,1359,216,'Flame_Skimmer',900,0,842,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (51,276,216,'Athamas',900,0,186,0,0,90,90,0);
-INSERT INTO `mob_groups` VALUES (52,2302,216,'Kutharei',0,128,1471,0,999,95,95,0);
-INSERT INTO `mob_groups` VALUES (53,2095,216,'Ironclad_Severer',900,0,1372,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (54,3798,216,'Sturdy_Pyxis',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (55,3813,216,'Surveyor',300,0,0,0,0,0,0,0); -- TODO: capture level and respawn time from retail
+INSERT INTO `mob_groups` VALUES (20,1157,216,'Dynamo_Cluster',300,0,731,5316,0,82,83,0);
+INSERT INTO `mob_groups` VALUES (21,279,216,'Atrociraptor',300,0,189,6224,0,83,84,0);
+INSERT INTO `mob_groups` VALUES (22,1142,216,'Dusk_Lizard',300,0,723,5900,0,83,84,0);
+INSERT INTO `mob_groups` VALUES (23,2567,216,'Maritime_Peiste',300,0,1621,4581,0,84,85,0);
+INSERT INTO `mob_groups` VALUES (24,26,216,'Abyssobugard',300,0,6,5496,0,84,85,0);
+INSERT INTO `mob_groups` VALUES (25,2992,216,'Orapodium',300,0,1870,4543,0,84,85,0);
+INSERT INTO `mob_groups` VALUES (26,2139,216,'Jala',0,128,0,0,0,0,0,0); --
+INSERT INTO `mob_groups` VALUES (27,4007,216,'Tristitia',0,128,2472,59998,0,90,90,0);
+INSERT INTO `mob_groups` VALUES (28,2781,216,'Mxghrah',0,128,0,32485,0,87,88,0);
+INSERT INTO `mob_groups` VALUES (29,2682,216,'Mighrah',0,128,0,4162,0,84,85,0);
+INSERT INTO `mob_groups` VALUES (30,2666,216,'Minax_Bugard',0,128,1694,36015,0,87,88,0);
+INSERT INTO `mob_groups` VALUES (31,3635,216,'Sirrush',0,128,2258,30348,0,87,88,0);
+INSERT INTO `mob_groups` VALUES (32,1437,216,'Funereal_Apkallu',0,128,916,31416,0,87,88,0);
+INSERT INTO `mob_groups` VALUES (33,2553,216,'Manohra',0,128,1615,30229,0,87,88,0);
+INSERT INTO `mob_groups` VALUES (34,678,216,'Cep_Kamuy',0,128,445,30037,0,87,88,0);
```

Comment:
the dash is actually valid

Reply:
also just a heads up but most mobs in abyssea can have a much wider ranged of leveled based on how much they are being curb stomped by players. then when left alone long enough they start de-leveling back to normal. when we do caps we're going to want to tray and catch them at their lowest possible base levels to avoid unintended skewing 👍

Reply:
Yea all  Non NM type are level 77-95,  i did not add them because, i didn't know if the system worked where if  you leave them alone for a certain they respawned as level 77-79.